sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -588,6 +595,14 @@ feature, the application must be compiled in ahead-of-time (AOT) mode using
 "special target values" listed in the link:../../UsersManual.md[users manual]
 description of the `-fsycl-targets` option.  These are the target names of the
 form "intel_gpu_*", "nvidia_gpu_*", or "amd_gpu_*".
+The architecture enumeration `intel_cpu_spr` does not currently work
```

Comment:
Please add a paragraph break (blank line) before this sentence.  The limitation about `intel_cpu_spr` should be in its own paragraph.

Reply:
done